.eleventy.js

```diff
@@ -1,5 +1,5 @@
 module.exports = eleventyConfig => {
-  eleventyConfig.addPairedShortcode("postcss", require("./utils/transform-css"));
+  eleventyConfig.addFilter("postcss", require("./utils/transform-css"));
```

Comment:
My gut is telling me that Liquid/Nunjucks really doesn't like async filters.
Will reply in the 11ty repo for better visibility/discussion.